.env.electron

```
@@ -0,0 +1,28 @@
+# Electron Environment Variables
+# This file is for ELECTRON BUILDS ONLY and should NOT be committed to repository
+# Copy from .env.example and add your actual electron-specific configuration
+
+# Application URLs (Electron-specific)
+APP_URL=http://localhost:4200/
+API_URL=https://apitest.giddh.com/
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_URL key should go before the APP_URL key

(UnorderedKey)